src/physics/em/detail/SBEnergyDistribution.hh

```diff
@@ -0,0 +1,135 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file SBEnergyDistribution.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "physics/base/Units.hh"
+#include "physics/grid/TwodSubgridCalculator.hh"
+#include "random/distributions/UniformRealDistribution.hh"
+#include "SeltzerBerger.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Sample exiting photon energy from Bremsstrahlung.
+ *
+ * The SB energy distribution uses tabulated Seltzer-Berger differential cross
+ * section data, which stores cumulative probabilities as a function of
+ * incident particle energy and exiting gamma energy (see SeltzerBergerModel
+ * for details). The sampling procedure is roughly laid out in section
+ * [PHYS341] of the GEANT3 physics reference manual, although like Geant4 we
+ * use raw tabulated  SB data rather than a parameter fit. Also like Geant4 we
+ * include the extra density correction factor.
+ *
+ * The algorithm is roughly thus. Calculate allowable exit energy range from
```

Comment:
thus -> this

Reply:
Actually correct as is -- the usage is slightly formal.
> in the manner now being indicated or exemplified; in this way

Reply:
Did not know you could use it this way. Good to know...